src/components/SearchContainer/SearchContainer.tsx

```diff
@@ -15,10 +15,7 @@ function SearchContainer({ handleIncrement, handleSubmit }: any) {
   }
 
   return (
-    <form
-      className="searchContainer"
-      onSubmit={updateSearch()}
-    >
+    <form className="searchContainer" onSubmit={updateSearch()}>
```

Comment:
I believe this function can be updated to `onSubmit={updateSearch}` and
```
function updateSearch(event) {
      event.preventDefault();
      handleIncrement();
      handleSubmit(searchEntry);
  }
```

Reply:
Thank you. Will fix it